README.md

```diff
@@ -35,6 +37,24 @@ on macOS/Linux
 
     `pip install fastapi mongodb uvicorn`
```

Comment:
unsure about step 2, all dependencies should be in requirements file and step1 should be enough

Reply:
I deleted it